service/download_model.py

```diff
@@ -0,0 +1,25 @@
+import sys
+
+# # Check if two command-line arguments are provided
+if len(sys.argv) !=3:
+    print("Usage: python download_model.py <whisper_model_id> <whisper_model_output_path>")

```

Comment:
here as well think if we could use logger
also the print messages doesn't actually informative